### ISSUE TITLE
Automate backport labels update

### ISSUE THREAD

**Issue by aditighag** [cilium/cilium#16657]
It's quite common for Janitors to forget updating the backport labels once a backport PR is merged. This is an aspirational issue that I'm looking to tackle while I'm working on https://github.com/cilium/cilium/issues/16018. 🤞 

Edit : Looked into this, but I was unable to find the right GH APIs to achieve this.

**Comment by aanm** [comment]
Duplicate of https://github.com/cilium/github-actions/issues/12
